content/docs/trust/trust-manager/README.md

```diff
@@ -177,22 +172,41 @@ to the trust-manager namespace by default.
 
 ## Quick Start Example
 
-Let's get started with an example of creating our own `Bundle`!
+Let's get started with an example of creating our first `Bundle`, using [KinD](https://kind.sigs.k8s.io/) to run Kubernetes in a container!
 
-First we'll create a demo cluster:
+First we'll create a cluster and a namespace:
 
 ```bash
-git clone https://github.com/cert-manager/trust-manager trust-manager
-cd trust-manager
-make demo
+kind create cluster
+# wait for cluster to be started...
 ```
 
-Once we have a running cluster, we can create a `Bundle` using the default CAs which were configured
-when trust-manager started up. Since we've installed trust-manager using Helm, our default CA package
+Next, we need to install trust-manager. You can follow the [installation guide](./installation.md) to do that,
+or use the commands below:
+
+```bash
+helm repo add jetstack https://charts.jetstack.io --force-update
+
+helm install cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version [[VAR::cert_manager_latest_version]] \
+  --set crds.enabled=true
+
+helm upgrade trust-manager jetstack/trust-manager \
+  --install \
+  --namespace cert-manager \
+  --wait
+```
+
+Once trust-manager is running we can create a `Bundle` using the default CAs which were configured
+when trust-manager started up.
+
+Since we've installed trust-manager using Helm, our default CA package
 contains publicly trusted certificates derived from a Debian container.
 
 ```bash
-kubectl --kubeconfig ./bin/kubeconfig.yaml apply -f - <<EOF
+kubectl apply -f - <<EOF
```

Comment:
I agree to this simplification, but there must have been a reason for the more complex command.

Reply:
These were all originally written assuming you'd run `make demo` in the trust-manager repo which put the kubeconfig in `bin/kubeconfig.yaml` (and didn't add it to `~/.kube/config`).

The makefile-modules migration broke `make demo` completely.

Now this guide tells us to use `kind create cluster` which will add the kubeconfig to `~/.kube/config` so it'll be there by default, so we don't need to tell people to pass the `--kubeconfig` flag.